### PR TITLE
fix(form-core): form-group should have a valid formattedValue

### PR DESCRIPTION
**Pull request by daKmoR**
closes https://github.com/ing-bank/lion/issues/859

still working on it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "build:docs": "wca analyze \"packages/tabs/**/*.js\"",
     "build:types": "tsc -p tsconfig.build.types.json",
     "bundlesize": "rollup -c bundlesize/rollup.config.js && bundlesize",
+    "debug": "web-test-runner \"packages/form-core/test/**/*.test.js\" --watch",
```
Comment (daKmoR) [inline package.json:12]: added this so you can always change it to which package you are currently working on...
later I wanna make it a real command so you can do 
```
yarn debug -g form-core
yarn debug -g overlays
...
```

requires some changes in web-test-runner so postponing this until then

Comment (tlouisse) [reply package.json:12]: so now only form-core supported on purpose?

File: packages/form-core/test/choice-group/ChoiceGroupMixin.test.js [hunk 2]
```diff
@@ -121,6 +136,19 @@ describe('ChoiceGroupMixin', () => {
     expect(el.formElements[2].checked).to.be.true;
   });
 
+  it('can set initial formattedValue on creation', async () => {
```
Comment (tlouisse) [inline packages/form-core/test/choice-group/ChoiceGroupMixin.test.js:139]: should we also check with `multipleChoice`?

Comment (daKmoR) [reply packages/form-core/test/choice-group/ChoiceGroupMixin.test.js:139]: added 👍

File: package.json (hunk 1 above)
Comment (daKmoR) [reply package.json:12]: right now I edit `form-core` to what I am working on in the `package.json`

Comment (daKmoR) [reply package.json:12]: which is of course a workaround 😅

**Pull request merged**